frontend/src/metabase/parameters/components/Parameters/syncQueryParamsWithURL.js

```diff
@@ -0,0 +1,75 @@
+import Dimension from "metabase-lib/lib/Dimension";
+
+export const syncQueryParamsWithURL = props => {
```

Comment:
Not the scope of this PR but for a subsequent follow-up, it would be nice to add a unit test for this (since now this function is exported).

Reply:
Agreed, will add them in a follow-up PR.